WordPress/gutenberg
# Site Editor Details text update to "Quick Edit"

## What?
Fixes https://github.com/WordPress/gutenberg/issues/66612

## Why?
The term 'Details' is not matching with the context. So updating it to 'Quick Edit'.

## How?
Changing the text 'Details' to 'Quick Edit'.

## Testing Instructions

- Go to the WordPress admin > Gutenberg > Experiments and enable the 'Quick Edit in DataViews' experiment.
- Go to the Site editor > Pages.
- Switch to the Table view or Grid view.
- In the data views top bar, observe the label of the button to toggle this panel will now display as 'Quick Edit'.
- Open the panel.
- Observe there's no heading at the top of the panel to communicate what this panel is about.


## Screenshots
|Before|After|
|-|-|
|![Screenshot 2024-12-18 at 1 49 32 PM (2)](https://github.com/user-attachments/assets/9b6a1f84-f935-4f1d-a01f-8e094cef439b)|<img width="187" alt="Screenshot 2024-12-18 at 1 51 33 PM" src="https://github.com/user-attachments/assets/d2a3fe6f-f221-4e86-811c-da08626d3d26" />|
